test/kill_node.sh

```diff
@@ -3,9 +3,15 @@
 # list of process name to be killed
 targetProcess=( "harmony" "bootnode" "soldier" "commander" "profiler" )
 
+NAME="$(uname -s)"
+
 for pname in "${targetProcess[@]}"
 do
-  sudo pkill -9 -e "^(${pname})$"
+  if [ $NAME == "Darwin" ]; then
+    pkill -9 "^(${pname})$"
+  else
+    sudo pkill -9 -e "^(${pname})$"
```

Comment:
I think this was introduced by @GheisMohammadi but it should be removed. Rather, the user should call `sudo ./test/kill_node.sh` instead. 

```suggestion
    pkill -9 -e "^(${pname})$"
```

Reply:
@Frozen can you please change before we merge?

Reply:
fixed